core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -1140,14 +1165,17 @@ support.role=Role
 support.agent=Support agent
 support.state=State
 support.chat=Chat
-support.requested=Requested
 support.closed=Closed
 support.open=Open
 support.process=Process
-support.buyerMaker=XMR buyer/Maker
-support.sellerMaker=XMR seller/Maker
+support.buyerOfferer=XMR buyer/Maker
```

Comment:
Needs reverted.

Reply:
Needs reverted.

Reply:
As shown in my workspace (without any changes):

![Screenshot from 2023-02-28 16-08-37](https://user-images.githubusercontent.com/20237127/221894589-105a38e8-d999-4819-888d-444238d7e619.png)

Reply:
Already handled above. Refer to duplicate above.

Reply:
For the "issue" below...once again without any changes in my workspace:

![Screenshot from 2023-02-28 16-32-12](https://user-images.githubusercontent.com/20237127/221901611-9268d97f-6854-47bb-ad10-51b0407425f7.png)